.github/workflows/publish.yml

```diff
@@ -313,7 +313,14 @@ jobs:
       VECTOR_VERSION: ${{ needs.generate-publish-metadata.outputs.vector_version }}
     strategy:
       matrix:
-        container: ["ubuntu:16.04","ubuntu:18.04","ubuntu:20.04","ubuntu:22.04","debian:9","debian:10","debian:11"]
+        container:
+          - ubuntu:14.04
```

Comment:
Intentionally added this one too? Wasn't mentioned in the description.

Reply:
Ah, yes, I forgot to note that in the description. 14.04 is still supported and we now support it again with the older libc linking.